### PR TITLE
workspace attributes updated the same way as entity attributes

### PR DESCRIPTION
**Pull request by dvoet**

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala [hunk 1]
```diff
@@ -108,6 +109,37 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
...
+    nickname = "update workspace",
+    httpMethod = "Patch",
+    produces = "application/json",
+    response = classOf[Workspace])
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
+    new ApiImplicitParam(name = "workspaceName", required = true, dataType = "string", paramType = "path", value = "Workspace Name"),
+    new ApiImplicitParam(name = "entityType", required = true, dataType = "string", paramType = "path", value = "Entity Type"),
+    new ApiImplicitParam(name = "entityName", required = true, dataType = "string", paramType = "path", value = "Entity Name"),
+    new ApiImplicitParam(name = "entityUpdateJson", required = true, dataType = "org.broadinstitute.dsde.rawls.workspace.EntityUpdateOperations$EntityUpdateOperation", paramType = "body", value = "Update operations")
```
Comment (jmthibault79) [inline src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala]: Missed a rename: should be `org.broadinstitute.dsde.rawls.workspace.AttributeUpdateOperations.AttributeUpdateOperation`

The Swagger for `updateEntityRoute` also needs this fix.

File: src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala [hunk 2]
```diff
@@ -108,6 +109,37 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
...
+  @Path("/{workspaceNamespace}/{workspaceName}")
+  @ApiOperation(value = "Update attributes of a workspace",
+    nickname = "update workspace",
+    httpMethod = "Patch",
+    produces = "application/json",
+    response = classOf[Workspace])
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
+    new ApiImplicitParam(name = "workspaceName", required = true, dataType = "string", paramType = "path", value = "Workspace Name"),
+    new ApiImplicitParam(name = "entityType", required = true, dataType = "string", paramType = "path", value = "Entity Type"),
```
Comment (jmthibault79) [inline src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala]: Copy/paste: `entityType` and `entityName` are not used here.

File: src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala (hunk 1 above)
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala]: I will fix in a subsequent PR dealing with apis

File: src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala [hunk 3]
```diff
@@ -317,21 +349,21 @@ class WorkspaceService(dataSource: DataSource, workspaceDAO: WorkspaceDAO, entit
...
+  case class AddListMember(attributeListName: String, newMember: Attribute) extends AttributeUpdateOperation
+  case class RemoveListMember(attributeListName: String, removeMember: Attribute) extends AttributeUpdateOperation
 
   private val AddUpdateAttributeFormat = jsonFormat2(AddUpdateAttribute)
   private val RemoveAttributeFormat = jsonFormat1(RemoveAttribute)
   private val AddListMemberFormat = jsonFormat2(AddListMember)
   private val RemoveListMemberFormat = jsonFormat2(RemoveListMember)
 
-  implicit object EntityUpdateOperationFormat extends RootJsonFormat[EntityUpdateOperation] {
+  implicit object EntityUpdateOperationFormat extends RootJsonFormat[AttributeUpdateOperation] {
```
Comment (jmthibault79) [inline src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala]: Should be renamed for consistency

File: src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala (hunk 2 above)
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala]: I will fix in a subsequent PR dealing with apis

**Pull request merged**